$NetBSD: patch-setup.py,v 1.1 2021/11/25 19:33:58 bsiegert Exp $

Use setuptools, distutils does not support the options we use

--- setup.py.orig	1970-01-01 00:00:00.000000000 +0000
+++ setup.py
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # setup.py generated by flit for tools that don't yet use PEP 517
 
-from distutils.core import setup
+from setuptools import setup
 
 packages = \
 ['confuse']
